bazel/repository_locations.bzl

```diff
@@ -162,12 +162,12 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "Abseil",
         project_desc = "Open source collection of C++ libraries drawn from the most fundamental pieces of Google’s internal codebase",
         project_url = "https://abseil.io/",
-        version = "c8b33b0191a2db8364cacf94b267ea8a3f20ad83",
-        sha256 = "a7803eac00bf68eae1a84ee3b9fcf0c1173e8d9b89b2cee92c7b487ea65be2a9",
+        version = "1adf896ec842bd9788a1bbede94a33e1402b8ecb",
```

Comment:
is there a reason we cant use release versions?

Reply:
abseil "lives at head" and we don't seem to use LTS branches. See https://abseil.io/about/releases.

Reply:
not sure im following this exactly - isnt that how all repos work ?

their release strategy seems a bit confusing - but afaict the "LTS branches" are the releases

if there is a reason not to use the releases - then i think its fine to use HEAD but generally its preferred to use published releases where possible/available

cc @htuch @moderation

Reply:
FWIW grpc does use abseil LTS releases https://github.com/grpc/grpc/blob/master/bazel/grpc_deps.bzl#L344C9-L344C9. I'd be fine instituting this as long as we're OK with some delay in picking up changes. I see that protoc uses some internal abseil detail that don't compile on every version.

Reply:
It's fine to use LTS releases by default and bump to HEAD as needed. I think this break glass might be needed as people use new features and we wouldn't artifically push back on that. This is consistent with other dependencies.